svthalia/concrexit
# Add promotion publish reminder

Closes #3340



### Summary
The website will send an email every morning to configurable email addresses per PromotionChannel, listing promotion requests that are scheduled to be published that day.

### How to test
Steps to test the changes you made:
1. Ensure there is a promotion request scheduled to be published today
2. `poetry run python website/manage.py sendpromodailyoverview`
3. Ensure that the correct mail is sent (check in terminal output)
